### PR TITLE
Support netstandard 2.0

### PR DESCRIPTION
**Pull request by John0King**
Now NPOI support netstandard 2.0,
So I update this lib to support netstandard 2.0 too

now #37  can be answered as yes

### CODE REVIEW

**Comment by John0King** [comment]
@donnytian  NPOI.Mapper is targeting `.NETStandard 2.0` , Only test are targeting `.netcoreapp2.1`

The `.NetStandard` is a library format, It's not suitable for test project

**Comment by John0King** [comment]
@donnytian Are you meaning the `System.Reflect.Emit` not available on IOS and UWP ?
a) We can say this library not support on Jit disabled platform.( `System.Reflect.Emit` itself is a .netstandard package  but not support on IOS and UWP either.beside , it will available for Xamarin.Android ) 
b) we can use `System.Reflect` API when Emit is not available for now. 
b) [the new mono-interpreter](https://www.mono-project.com/news/2017/11/13/mono-interpreter/) will enable `System.Reflect.Emit` in the future

**Comment by John0King** [comment]
@donnytian  Will this be merged ?

**Comment by John0King** [comment]
@donnytian  I update this PR a few days ago, I remove a dependency when target .net4.5, so it will be same like before (only npoi version changed, but npoi it doesn't have any nuget dependency eight when target .net4.5) except it now can also target .net standard (eg.  .net Core ，Xamarin.Android, mono)

so is there any reason that you can not merge this PR? or Is there anything you want me to change ?

File: Npoi.Mapper/src/Npoi.Mapper/AnonymousTypeFactory.cs [hunk 1]
```diff
@@ -19,8 +19,17 @@ public static class AnonymousTypeFactory
         static AnonymousTypeFactory()
         {
             var assemblyName = new AssemblyName { Name = "MyAnonymousTypes" };
-            var assemblyBuilder = System.Threading.Thread.GetDomain().DefineDynamicAssembly(assemblyName, AssemblyBuilderAccess.Run);
+
+            //var assemblyBuilder = System.Threading.Thread.GetDomain().DefineDynamicAssembly(assemblyName, AssemblyBuilderAccess.Run);
+            //ModuleBuilder = assemblyBuilder.DefineDynamicModule(assemblyName.Name);
```
Comment (donnytian) [inline Npoi.Mapper/src/Npoi.Mapper/AnonymousTypeFactory.cs]: This code contains commented out code. Consider removing them.

File: Npoi.Mapper/src/Npoi.Mapper/AnonymousTypeFactory.cs [hunk 2]
```diff
@@ -302,7 +311,9 @@ private static Type CreateGenericTypeDefinitionNoLock(string genericTypeDefiniti
             ).ToArray();
             DefineToStringMethod(typeBuilder, fieldPairs);
 
-            return typeBuilder.CreateType();
+
+            //return typeBuilder.CreateType();
```
Comment (donnytian) [inline Npoi.Mapper/src/Npoi.Mapper/AnonymousTypeFactory.cs]: This code contains commented out code. Consider removing them.

**Pull request merged**